de-de/einrichtung.md

```diff
@@ -0,0 +1,67 @@
+# Einrichtung des Modpacks
+
+**Magst du Video-Tutorials lieber als Text?** [**Klicke hier!**](https://github.com/Fabulously-Optimized/fabulously-optimized#reviews)
```

Comment:
May want to mention that the videos are in English or if you don't expect the audience to understand English (speech), you can remove it altogether

Reply:
:+1: Alright. I'm adding a notice.